connector/connector.js

```diff
@@ -70,3 +70,35 @@ connector.createMiddleware = function(phase, data, cb) {
   middlewareDef,
   cb);
 };
+
+connector.createModel = function(id, data, cb) {
```

Comment:
Something weird about the indentation and contents of this file, is there a module.exports I'm missing?